src/firebase.android.ts

```diff
@@ -1335,6 +1335,10 @@ firebase.updateProfile = arg => {
   });
 };
 
+ /***********************************************
```

Comment:
Yeah, I still have to extract these Realtime DB functions to their own module one day 😊